runtime/doc/provider.txt

```diff
@@ -112,6 +112,32 @@ To use an absolute path (e.g. to an rbenv installation): >
 To use the RVM "system" Ruby installation: >
     let g:ruby_host_prog = 'rvm system do neovim-ruby-host'
 <
+==============================================================================
+Node integration				*provider-node*
+
+The node provider allows nvim to run remote plugin managed by nodejs, see
+https://nodejs.org/
+
+NODE QUICKSTART~
+
+To use javascript remote plugins with Nvim, just install the latest `neovim`
+ package: >
+    $ npm install -g neovim
+
+NODE PROVIDER CONFIGURATION~
+
+						*g:loaded_node_provider*
+To disable Node support: >
+    let g:loaded_node_provider = 1
+<
```

Comment:
Here you close the `helpExample`, but not in the other cases.

Reply:
fixed